charts/sda-svc/templates/finalize-deploy.yaml

```diff
@@ -71,7 +71,7 @@ spec:
         - name: BROKER_EXCHANGE
           value: {{ default "sda" .Values.global.broker.exchange }}
         - name: BROKER_QUEUE
-          value: "accessionIDs"
+          value: "accession"
```

Comment:
double check: if this has been changed at all places (`accessionIDs` -> `accession`)

Reply:
Mixed case queue names has been deprecated since a while Bach for rabbitMQ, we are only now catching up on that.